lib/stdlib/strncpy.c

```diff
@@ -0,0 +1,57 @@
+/*	$OpenBSD: strncpy.c,v 1.7 2014/06/10 04:17:37 deraadt Exp $	*/
```

Comment:
Please state in the commit message the precise repository location where you are importing this code from. All the other stdlib code in TF-A is from FreeBSD so it's preferable to use that repo.